src/main/java/com/splunk/ibm/mq/metricscollector/ChannelMetricsCollector.java

```diff
@@ -37,15 +35,20 @@ public final class ChannelMetricsCollector implements MetricsPublisher {
   private static final Logger logger = LoggerFactory.getLogger(ChannelMetricsCollector.class);
 
   public static final String ARTIFACT = "Channels";
-  private final MetricCreator metricCreator;
   private final MetricsCollectorContext context;
+  private final ChannelMetrics channelMetrics;
```

Comment:
I cannot find where this class is defined. I am on my phone so difficult to be sure. Was this class a new file?

Reply:
would you please `git add -A && git commit --amend && git push origin channel_otel_only` ?